examples/measure.js

```diff
@@ -286,21 +274,16 @@ var formatLength = function(line) {
 
 
 /**
- * format length output
+ * format area output
  * @param {ol.geom.Polygon} polygon
  * @return {string}
  */
 var formatArea = function(polygon) {
-  var area;
-  if (geodesicCheckbox.checked) {
-    var sourceProj = map.getView().getProjection();
-    var geom = /** @type {ol.geom.Polygon} */(polygon.clone().transform(
-        sourceProj, 'EPSG:4326'));
-    var coordinates = geom.getLinearRing(0).getCoordinates();
-    area = Math.abs(wgs84Sphere.geodesicArea(coordinates));
-  } else {
-    area = polygon.getArea();
-  }
+
+  var coordinates = polygon.getLinearRing(0).getCoordinates();
+  var proj = map.getView().getProjection();
+  var area = ol.proj.getArea(coordinates, proj);
```

Comment:
If `getArea()` received an `{Array.<Array.<ol.Coordinate>>}`, it would also work for polygons with holes and you wouldn't have to do `getLinearRing(0)` above.